lib/src/functions/math.dart

```diff
@@ -98,3 +101,57 @@ BuiltInCallable _numberFunction(String name, num transform(num value)) {
         denominatorUnits: number.denominatorUnits);
   });
 }
+
+final _clamp = BuiltInCallable("clamp", r"$min, $preferred, $max", (arguments) {
```

Comment:
`$preferred` is a different parameter name than what the spec says (`$number`).

Reply:
Good catch, thanks.